### PR TITLE
Minor clean up, addition of new sections, and update/polish checklist

### PR DESCRIPTION
**Pull request by davidlmobley**
- Fixes a bib item which had a bad unicode character
- Removes a to-do item which was resolved 
- Moves the checklist up
- Adds conclusions
- Adds section on whether you should run MD at all
- Adds section on avoiding overinterpreting MD
- Updates/polishes checklist

### CODE REVIEW

**Comment by davidlmobley** [comment]
Added a section on whether you should run MD. The answer is not always "yes". Check that part, @JIMonroe ?

More still to come later, but I'll be away for an hour or so.

File: paper/basic_training.tex [hunk 1]
```diff
@@ -1211,10 +1197,51 @@ \subsubsection{Grid based Ewald summation}
...
+
+\section{Should you run MD?}
+
+A critical question \emph{before} preparing an MD simulation of your system is whether you even \emph{should} use MD for your system in view of the resources you have and what information you hope to obtain.
+MD is a tool, but it may not be the right tool for your problem.
+Before beginning any study, it is critical to sort out what questions you want to answer, what resources (computational and otherwise) you have at your disposal, and whether you have any information about your system(s) of interest that indicate you can realistically expect to answer those questions given a set of MD simulations.
+
+As noted above, the frequency of the fastest vibrational motions in a system of interest sets a fundamental limit on the timestep which, given fixed computational resources, sets a limit on how much simulation time can be covered with any reasonable amount of computer time.
+Thus, as noted in the Introduction, the longest all-atom MD simulations are on the microsecond to millisecond timescale.
```
Comment (JIMonroe) [inline paper/basic_training.tex]: Should tag introduction (\ref{sec:Introduction}) if we get the chance.

File: paper/basic_training.tex [hunk 2]
```diff
@@ -1211,10 +1197,51 @@ \subsubsection{Grid based Ewald summation}
...
+This means that if your system is complex and answering your questions will require sampling critical events that have a timescale of seconds or longer, MD will not be the right tool for the job.
+You could invest a huge amount of effort running a MD simulations and find that these do nothing whatsoever to address your questions.
+
+Ideally, you should have some information before beginning that the relevant timescales for your system might be accessible given the amount of MD you can afford to run, or you could plan a set of exploratory MD simulations to assess feasibility. 
+But don't simply plunge in and attempt to run simulations until you find the answers to your questions, as the required timescales could end up being orders of magnitude longer than what you can afford to spend.
```
Comment (JIMonroe) [inline paper/basic_training.tex]: I agree with what you say completely, but from here onwards we may be able to condense a little bit and also perhaps adjust the tone.

File: paper/basic_training.tex [hunk 3]
```diff
@@ -1211,10 +1197,51 @@ \subsubsection{Grid based Ewald summation}
...
+
+So, count the cost of your potential simulations, assess whether they realistically have a chance of answering the questions you seek to answer, and then carefully decide whether the likelihood of success is worth the cost. 
+If not, don't tackle that problem with MD.
+
+
+\todo[inline, color={yellow!20}]{DLM: Perhaps also a brief ``what NOT to do with your MD data'' blurb, e.g., don't just make movies and look at them. Don't treat them as the answer. Don't overinterpret, etc.}
+
 
+\section{Conclusions}
+Molecular simulations are particularly exciting, because they provide a detailed view into the structure and function of systems at a molecular or atomistic level, and allow us to precisely compute thermodynamic and statistical properties and connect these to the underlying motions, structure, and function.
```
Comment (JIMonroe) [inline paper/basic_training.tex]: Check sentence structure here (making picky notes since we'll only have so much time for revisions at this point).

Comment (davidlmobley) [reply paper/basic_training.tex]: I think it's grammatically correct, but would you prefer two sentences?

Comment (JIMonroe) [reply paper/basic_training.tex]: Right, it's just a little awkward.  And maybe remove "the" before underlying... ?

**Comment by davidlmobley** [comment]
@JIMonroe - I just pushed another new section resolving another of my to-dos. Can you edit while I address the issues you raised above?

File: paper/basic_training.tex (hunk 1 above)
Comment (davidlmobley) [reply paper/basic_training.tex]: Done

File: paper/basic_training.tex (hunk 2 above)
Comment (davidlmobley) [reply paper/basic_training.tex]: I'll let you attempt to adjust tone in a PR unless you have specific suggestions. Perhaps I wrote slightly informally, though I tend to prefer "we should" (or "us", etc.) to "one should", ESPECIALLY for introductory material, as it has more of a feel of bringing the reader along for the ride. I should probably avoid contractions like "don't" though.

**Comment by davidlmobley** [comment]
@JIMonroe OK I think this is ready to merge.

**Comment by JIMonroe** [comment]
I was in meetings for the past couple hours, so I'm just seeing all this now.

The edits look great!  The section on interpreting MD results seems very helpful.

As for adding info on how often to save snapshots, you're list looked good.  I might also add that sometimes you don't know the correlation times until you've collected the data (if you're working with some new metric for instance).  Still, we should try and keep this brief, as other documents should in principle also cover this.

I'll merge in and any other edits can be in separate PRs.

**Pull request merged**